nni/algorithms/compression/v2/pytorch/pruning/movement_pruner.py

```diff
@@ -171,9 +155,9 @@ def trainer(model: Module, optimizer: Optimizer, criterion: Callable[[Tensor, Te
                     # If you don't want to update the model, you can skip `optimizer.step()`, and set train mode False.
                     optimizer.step()
                 model.train(mode=training)
-    optimizer : torch.optim.Optimizer
-        The optimizer instance used in trainer. Note that this optimizer might be patched during collect data,
-        so do not use this optimizer in other places.
+    traced_optimizer : nni.common.serializer.SerializableObject(torch.optim.Optimizer)
```

Comment:
There is an interface called `Traceable`. `SerializableObject` is only its concrete implementation,

Reply:
fixed